tests/test_reader.py

```diff
@@ -181,13 +188,17 @@ def test_reader_after_crash(mds_dataset_dir: Any, created_ago: float, download_t
 )
 @pytest.mark.usefixtures('mds_dataset_dir')
 @pytest.mark.parametrize('index', [17])
-def test_reader_getitem(mds_dataset_dir: Any, share_remote_local: bool, index: int):
+@pytest.mark.parametrize('seed', [5566])
```

Comment:
is all this seed setting because every test has to use a different seed or has conflicts?

Reply:
no conflicts. Added seed value to the tests for sample order randomness.